sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Data-Quality-With-SageMaker-Pipelines-On-Demand.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>Nit: run-on sentence - split up "To do this, we use the <code>suggest_baseline</code> method, the purpose of this is to generate a set of <code>statistics</code> and <code>constraints</code> file" into two sentences (To do this, we use the <code>suggest_baseline</code> method. The purpose of this is to generate a set of <code>statistics</code> and <code>constraints</code> file.)</p><p> </p><p>Please only use present tense (i.e. these files are used by). Make sure this applies to the rest of the notebook and other notebooks in this PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Doesn't seem to be addressed?</p>

Reply:
<p>Must address before merge!</p>

Reply:
Done